### DISCUSSION TITLE
[Feature] Product Key Purchase with Bitcoin

### DISCUSSION THREAD

**Discussion by jackedproxy** [immich-app/immich#13683] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Greetings,

I'm looking to purchase a product key to support the Immich project, however, I can't seem to find Bitcoin as a currency option (Can't find on-chain, nor via Lightning Network).

I think it's very much worth considering as Immich is a global project which could benefit from global donations, whereby a lot of people in the world do not have a bank account and/or are considered unbankable, but can easily have a Bitcoin wallet. 

Regards.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
The Immich team don't have direct control over payment methods, please email billing@futo.org to share your interest there.
